service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/ReturnedCaseContentProvider.java

```diff
@@ -36,7 +36,7 @@ private ReturnedCaseTemplate.ReturnedCaseTemplateBuilder templateData(CaseData c
         ReturnApplication returnApplication = caseData.getReturnApplication();
 
         return ReturnedCaseTemplate.builder()
-            .localAuthority(laLookup.getLocalAuthorityName(caseData.getCaseLocalAuthority()))
+            .localAuthority(caseData.getApplicantName().orElse(null))
```

Comment:
wonder if this should default to "The applicant" (based on the template?), very unlikely but maybe a useful default!